README.md

```diff
@@ -0,0 +1,25 @@
+# The Dollar Effect
+
+A subscription system to sponsor a non-profit organization of your choice for $1 a month.
+
+## Getting Started
+
+To be able to make changes in the repository, you need to be added to the github of dollareffect.github.io.
+Right now, only Albert can add members. Tag him in the project text channel so he can add you.
+
+### Prerequisites
+
+1. Text Editor ([Visual Studio Code](https://code.visualstudio.com), [Sublime Text](https://www.sublimetext.com))
```

Comment:
Let's remove text editor as prerequisites

Reply:
I see. I second your comment. Thank you!